content/docs/connect/choose-connection.md

```diff
@@ -0,0 +1,163 @@
+
+**Direct connections**:
+
+1. Install the Postgres client library (e.g., pg for Node.js).
+
+   ```bash
+   npm install pg dotenv
+   ```
+
+1. Set up the connection string in your application configuration.
+
+   ```javascript
+   const { Client } = require('pg');
+   const client = new Client({
+     connectionString: process.env.DATABASE_URL,
+   });
+   client.connect();
+   ```
+
+1. Use the client for database operations.
+
+   ```javascript
+   client.query('SELECT * FROM users', (err, res) => {
+     if (err) throw err;
+     console.log(res.rows);
+     client.end();
+   });
+   ```
+
+**Pooled Connections**:
+
+1. You don't need install PgBouncer; just use the `-pooler` suffix with your connection string.
+1. Modify the connection string to use this pooled endpoint.
+
+   ```javascript
+   const { Pool } = require('pg');
+   const pool = new Pool({
+     connectionString: process.env.DATABASE_URL_POOLER,
+   });
+   ```
+
+1. Use the pool for database operations.
+
+   ```javascript
+   pool.query('SELECT * FROM users', (err, res) => {
+     if (err) throw err;
+     console.log(res.rows);
+     pool.end();
+   });
+   ```
+
+**Serverless Driver**:
+
+1. Install the serverless driver.
+
+   ```bash
+   npm install @neondatabase/serverless
+   ```
+
+1. Configure the connection for HTTP or WebSocket.
+
+   ```javascript
+   const { Client } = require('@neondatabase/serverless');
+   const client = new Client({
+     connectionString: process.env.DATABASE_URL,
+   });
+   client.connect();
+   ```
+
+1. Use the client for database operations.
+
+   ```javascript
+   client.query('SELECT * FROM users', (err, res) => {
+     if (err) throw err;
+     console.log(res.rows);
+     client.end();
+   });
+   ```
+
+## Monitoring
+
+- **Use pg_stat_statements**: Use the `pg_stat_statements` extension to track information like execution counts and slow queries to help understand your query performance.
+
+  See [pg_stats_statements](docs/extensions/pg_stat_statement) for more information, in particular the [Monitor slow queries](/docs/extensions/pg_stat_statements#monitor-slow-queries) section.
+
+- **Monitoring Dashboard**: Use the [Monitoring Dashboard](docs/introduction/monitoring-page) in the Neon Console to monitor connection metrics, compute usage, and database health.
+
+- **External tools**: Use external tools like PgHero and pgAdmin to monitor your database. See [Monitoring Neon with external tools](docs/introduction/monitor-external-tools) for information about setting up and using these tools.
+
+## Table summarizing your options
+
+Here is a table summarizing the options we've walked through on this page:
+
+| Feature                   | Direct Connections                                                                                   | Pooled Connections                                                                   | Serverless Driver (HTTP)                 | Serverless Driver (WebSocket)                 |
```

Comment:
The table is not showing up correctly in the output.

Reply:
fixed